### ISSUE TITLE
PyTorch Hugging Face Models do not have ACL calls for Docker versions > 23.05

### ISSUE THREAD

**Issue by abhishek-rn** [ARM-software/Tool-Solutions#200]
Hi,

Docker Tags:
 [r23.09-torch-2.0.0-onednn-acl](https://hub.docker.com/layers/armswdev/pytorch-arm-neoverse/r23.09-torch-2.0.0-onednn-acl/images/sha256-59a99ce449bbc79de31a2c9d20d0021c2db134db228a1624cd02ba7f994f8729?context=explore)
[r23.05-torch-2.0.0-onednn-acl](https://hub.docker.com/layers/armswdev/pytorch-arm-neoverse/r23.05-torch-2.0.0-onednn-acl/images/sha256-6d82d82fa1c433b4bf868c56c4a1e2a168edb164fb2b8304e852e43de32fa6af?context=explore)

I am unable to get acl calls in docker versions higher than 23.05 for Pytorch Hugging Face Models

Attaching oneDNN verbose calls for BERT model here
[23.05_Bert_Verbose.txt](https://github.com/ARM-software/Tool-Solutions/files/12656128/23.05_Bert_Verbose.txt)
[23.09_Bert_Verbose.txt](https://github.com/ARM-software/Tool-Solutions/files/12656129/23.09_Bert_Verbose.txt)

The code to reproduce this is attached as below:
[PyT_Bert_Training.txt](https://github.com/ARM-software/Tool-Solutions/files/12656136/PyT_Bert_Training.txt) --> Use this for the first run to generate necessary inference checkpoints and files.
[PyT_Bert_Inf.txt](https://github.com/ARM-software/Tool-Solutions/files/12656137/PyT_Bert_Inf.txt) --> For subsequent runs to generate the oneDNN logs

Also, as a result, the later oneDNN verbose exhibits gemm:jit calls for Matmuls and this results in poor performance for inference compared to gemm:acl calls.

Thanks

**Comment by nSircombe** [comment]
Hi @abhishek-rn 
Thanks for the report. This transition from 23.05 to 23.06 marks the move from PyTorch 1.x to 2.x, so it looks like we may have lost some functionality at this stage.
Would you be able to confirm if the same behaviour is present if you use the `pip install`ed pytorch packages for 1.3 and 2.0 on AArch64, and also on x86?

**Comment by nSircombe** [comment]
Yes you're right, the version is 2.0. The tag is correct - matches the version in the Dockerfile. The mistake is in the README for the 23.05 increment here which still has 1.3.
